tests/ml/pytorch/test_pytorch_model_upload_pytest.py

```diff
@@ -103,17 +103,27 @@ def download_model_and_start_deployment(tmp_dir, quantize, model_id, task):
 
 
 class TestPytorchModel:
+    def __init__(self):
+        # quantization does not work on ARM processors
+        import platform
+
+        self.quantize = True if platform.machine() != "arm64" else False
```

Comment:
Does `platform.machine()` return `aarch64` on ARM Linux? (It will if it simply returns the output of `uname -m`.) If so, please change the condition to check for `platform.machine()` being either `arm64` or `aarch64`.